src/test/java/uk/gov/hmcts/reform/pip/account/management/service/AccountServiceTest.java

```diff
@@ -500,14 +511,38 @@ void testUploadMediaFromInvalidCsv() throws IOException {
             .getResourceAsStream("csv/invalidCsv.txt")) {
             MultipartFile multipartFile = new MockMultipartFile("file", "TestFileName",
                                                                 "text/plain",
-                                                                IOUtils.toByteArray(inputStream));
+                                                                IOUtils.toByteArray(inputStream)
+            );
 
             CsvParseException ex = assertThrows(CsvParseException.class, () ->
-                accountService.uploadMediaFromCsv(multipartFile, EMAIL),
-                                                "Should throw CsvParseException");
+                                                    accountService.uploadMediaFromCsv(multipartFile, EMAIL),
+                                                "Should throw CsvParseException"
+            );
             assertTrue(ex.getMessage().contains("Failed to parse CSV File due to"), MESSAGES_MATCH);
 
-
         }
     }
+
+    @Test
+    void testMiEndpoint() {
```

Comment:
Sorry just spotted the method name here....it is testing the service method not endpoint. Consider renaming.

Reply:
Fixed now :)